wagtail/wagtailsnippets/edit_handlers.py

```diff
@@ -35,6 +49,10 @@ def render_as_field(self):
             'snippet_type_name': self.snippet_type_name,
         }))
 
+    @property
+    def snippet_type_name(self):
+        return force_text(self.target_content_type()._meta.verbose_name)
```

Comment:
Actually, on second thoughts, we should just make `snippet_type_name` a class method instead. This isn't a public API, so there's no need to keep it as a property for the sake of backwards compatibility.